usage/camera/raspicam/raw.md

```diff
@@ -1,14 +1,14 @@
-# Using RAW on the Raspberry pi cameras
+# Using RAW on the Raspberry Pi cameras
 
 The definition of raw images can vary. The usual meaning is raw Bayer data directly from the sensor, although some may regard an uncompressed image that has passed through the ISP (and has therefore been processed) as raw.
 
-Both options are available from the Raspberry Pi Cameras.
+Both options are available from the Raspberry Pi cameras.
 
 ## Processed, non-lossy images
 
-The usual output from raspistill is a compressed JPEG file that has passed through all the stages of image processing to produce a high quality image. However, JPEG, being a lossy format does throw away some information that the user may want.
+The usual output from `raspistill` is a compressed JPEG file that has passed through all the stages of image processing to produce a high-quality image. However, JPEG, being a lossy format does throw away some information that the user may want.
 
-`raspstill` has an `encoding` option that allows you to specify the output format. Options are `jpg`, `bmp`, `gif` and `png`. The latter three are non-lossy, so no data is thrown away in an effort to improve compression, but do require conversion from the original YUV, and because these formats do not have hardware support they produce images slightly more slowly than JPEG.
+`Raspstill` has an `encoding` option that allows you to specify the output format: either `jpg`, `gif`, `bmp` or `png`. The latter two are non-lossy, so no data is thrown away in an effort to improve compression, but do require conversion from the original YUV, and because these formats do not have hardware support they produce images slightly more slowly than JPEG.
```

Comment:
Why the change from "three" to "two"? GIF, BMP and PNG are all non-lossy.

Reply:
> Why the change from "three" to "two"? GIF, BMP and PNG are all non-lossy.

GIF *is* lossy, since it has a maximum of 256 colours.

Reply:
Oh! Might be worth adding a few words explaining that?

Reply:
> Oh! Might be worth adding a few words explaining that?

I'm not sure that's necessary. My feeling it is would add unnecessary complication to the page. All folk really need to know is which formats are not lossy - if they want more information, they can find that information elsewhere quite easily.